Dockerfile

```diff
@@ -0,0 +1,91 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && \
+    apt-get install -y \
+    autoconf \
+    git \
+    curl \
+    libdrm-dev \
+    libgl1-mesa-glx \
+    libgl1-mesa-dev \
+    libtool \
+    libx11-dev \
+    openbox \
+    unzip \
+    xorg \
+    xorg-dev \
+    && apt-get clean all
+
+RUN mkdir /opt/src && \
+    cd /opt/src && \
+    curl -sSLO https://cmake.org/files/v3.8/cmake-3.8.2-Linux-x86_64.sh && \
+    sh cmake-3.8.2-Linux-x86_64.sh --prefix=/usr/local --exclude-subdir && \
+    rm cmake-3.8.2-Linux-x86_64.sh
+
+RUN cd /opt/src && \
+    curl -sSLO https://www.samba.org/ftp/ccache/ccache-3.2.8.tar.bz2 && \
+    tar xf ccache-3.2.8.tar.bz2 && \
+    cd ccache-3.2.8 && \
+    ./configure --prefix=/usr && \
+    make && \
+    make install
+
+RUN mkdir -p /usr/lib/ccache && \
+    cd /usr/lib/ccache && \
+    ln -sf /usr/bin/ccache gcc && \
+    ln -sf /usr/bin/ccache g++ && \
+    ln -sf /usr/bin/ccache cc && \
+    ln -sf /usr/bin/ccache c++ && \
+    ln -sf /usr/bin/ccache clang && \
+    ln -sf /usr/bin/ccache clang++ && \
+    ln -sf /usr/bin/ccache clang-4.0 && \
+    ln -sf /usr/bin/ccache clang++-4.0
+
+RUN cd /opt/src && \
+    git clone https://github.com/intel/libva && \
+    cd libva && \
+    git checkout 2.2.1.pre1-20180921 && \
```

Comment:
What's the purpose of this docker file? If that's to prepare _latest_ media environment, then why it is full of specific tags? If that's to reflect certain mediasdk stack release, then: 1) why that's in master branch, 2) why mediasdk tag is not used?

Reply:
Besides of quarterly release, we have other pre-release in weekly or bi-weekly cadence which is qualified by CI suite. This dockerfile is designed to create an automated build in hub.docker.com based on these pre-release tag, then user could docker pull the image directly.

Reply:
For mediasdk, I'm not sure which version is targeting to the build we verified 7.0 .0484-OS-OS1.02. Could you please provide the tag or commit ID?